thriftbp/client_pool.go

```diff
@@ -460,11 +460,11 @@ func newClient(
 		return nil, fmt.Errorf("thriftbp: error getting next address for new Thrift client: %w", err)
 	}
 
-	conn, err := net.DialTimeout("tcp", addr, cfg.GetConnectTimeout())
+	trans := thrift.NewTSocketConf(addr, cfg)
```

Comment:
I would avoid `trans` and go with something like `tsock` or `transport` or `xport`

Reply:
Changed every `trans` in the codebase to `transport`.